### PR TITLE
Allow more than one breadboard image to be specified

### PR DESCRIPTION
**Pull request by Resseguie**
Can add entries to `breadboards.json` if you want multiple breadboards for an example or to use an existing one with a different name. (Defaults to example name as before)

### CODE REVIEW

**Comment by Resseguie** [comment]
Fixes #695, minus moving the breadboard diagrams to a new sub-folder.

**Comment by Resseguie** [comment]
@rwaldron I made a single entry in breadboards.json to test with. I used led.js and added the simple diagram with LED direction in pin 13 and the one with a resistor. 

I'm not entirely happy with the markdown formatting for multiple diagrams. Because of the whitespace at the bottom of the image where the fritzing logo is, the .fzz link for the first image visually appears to go with the second diagram. I'm guessing I could fix that with some creative, but messy and non-obvious, markdown. Alternatively, the link on the image could point to the fzz (though I'm not a fan of that either).

I could change the simple breadboards.json format to use an object instead of an array, then you could assign a title to each diagram and have a header that visually separates them if multiples are given. Though it adds complexity to breadboards.json, that might solve the visual problem and give a way to describe each breadboard, especially once you get to your vision of one per platform.

thoughts? Or is this good enough for now?

File: Gruntfile.js [hunk 1]
```diff
@@ -207,8 +203,34 @@ module.exports = function(grunt) {
...
+          } else {
+            verbose.writeln("Missing PNG: " + png);
+          }
+
+          if (hasFzz) {
+            images += templates.fritzing({ fzz: fzz });
+          } else {
+            verbose.writeln("Missing FZZ: " + fzz);
+          }
+        });
```
Comment (rwaldron) [inline Gruntfile.js]: :white_check_mark:

File: Gruntfile.js [hunk 2]
```diff
@@ -218,8 +240,7 @@ module.exports = function(grunt) {
           example: eg,
           file: md,
           markdown: markdown.join("\n"),
-          breadboard: hasPng ? templates.img({ png: png }) : "",
-          fritzing: hasFzz ? templates.fritzing({ fzz: fzz }) : ""
+          breadboards: images.slice(0, -1)
```
Comment (rwaldron) [inline Gruntfile.js]: What is the last character?

File: docs/led.md [hunk 3]
```diff
@@ -31,8 +31,10 @@ board.on("ready", function() {
 ## Breadboard/Illustration
 
 
-![docs/breadboard/led.png](breadboard/led.png)
-[docs/breadboard/led.fzz](breadboard/led.fzz)
+![docs/breadboard/led-13.png](breadboard/led-13.png)  
+[Fritzing diagram: docs/breadboard/led-13.fzz](breadboard/led-13.fzz)
+![docs/breadboard/led-resistor.png](breadboard/led-resistor.png)  
+[Fritzing diagram: docs/breadboard/led-resistor.fzz](breadboard/led-resistor.fzz)
```
Comment (rwaldron) [inline docs/led.md]: :white_check_mark:

File: Gruntfile.js (hunk 2 above)
Comment (Resseguie) [inline Gruntfile.js]: Just an extra newline. I was trying to make it generate as close to the previous versions as possible.

**Pull request merged**